dashboard/test/ui/features/learning_platform/course_versions.feature

```diff
@@ -126,5 +126,5 @@ Scenario: Switch versions using dropdown on script overview page
 @as_student
 @no_mobile
 Scenario: Course unit family names redirect to their latest stable version
```

Comment:
Should the name of this test change to reflect that this will always go to 2019?

Reply:
Good catch! Also, I should change the test back so that it's actually following a redirect.